yarn-install/action.yml

```
@@ -50,6 +55,7 @@ runs:
 
     - name: yarn install
       shell: bash
+      working-directory: ${{ inputs.path }}
```

```suggestion
      working-directory: ${{ inputs.working-directory }}
```
To me this name of parameter makes more sense.

---

this would collide with the default `working-directory` `run` param and can be confusing, so actions use `path`. I think it makes sense to avoid such a confusion and still understand what's going on